testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -2623,6 +2626,54 @@ public void testInsertInTransaction()
         }
     }
 
+    @Test
+    public void testSelectAfterInsertInTransaction()
+    {
+        if (!hasBehavior(SUPPORTS_INSERT) || !hasBehavior(SUPPORTS_MULTI_STATEMENT_WRITES)) {
+            // nothing to test
+            log.info("Connector does not support insert in transaction context, so nothing to test");
```

Comment:
should we expect some exception here if we issue mulitple statements in transaction?
I expected it to be covered by some other test - but I did not find it when skimming.

Reply:
`testWriteNotAllowedInTransaction` seems to be doing that.